__tests__/appSpec.jsx

```diff
@@ -0,0 +1,50 @@
+import Adapter from 'enzyme-adapter-react-16';
+import { configure } from 'enzyme';
+import { App } from '../client/src/index';
+
+configure({ adapter: new Adapter() });
+
+describe('data tests', () => {
+  const app = new App();
+
+  test('title and map data should exist', () => {
+    app.getDummyData((err, title, maps) => {
+      expect(title.exists).to.equal(true);
+      expect(maps.exists).to.equal(true);
+    });
+  });
+
+  test('map data should have a length of 200', () => {
+    app.getDummyData((err, title, maps) => {
+      expect(maps).toHaveLength(200);
+    });
+  });
```

Comment:
You could possible combine these and just test map and title should have length of 200. It's up to your preference if you want to fix it.

Reply:
Nice, that makes sense.